compiler-core/src/type_.rs

```diff
@@ -459,6 +459,7 @@ pub struct ModuleInterface {
     pub types_constructors: HashMap<SmolStr, Vec<SmolStr>>,
     pub values: HashMap<SmolStr, ValueConstructor>,
     pub accessors: HashMap<SmolStr, AccessorsMap>,
+    pub unused: Vec<SrcSpan>,
```

Comment:
I think these are specifically unused imports?
```suggestion
    pub unused_imports: Vec<SrcSpan>,
```